eng/pipelines/runtimelab.yml

```diff
@@ -76,7 +76,7 @@ jobs:
     buildConfig: Release
     platforms:
     - Linux_x64
-    - Windows_NT_x64
+    - FreeBSD
     jobParameters:
```

Comment:
Do we want to run tests or just build at the moment?

Reply:
I wish we can run tests but I don't think we have any infrastructure with FreeBSD nodes. For build, we cross-compile on Linux.

Reply:
I see. Can't we add a `FreeBSD` docker image and run tests on that image?

Reply:
FreeBSD is not Linux. We would need FreeBSD host to run FreeBSD docker image.

Reply:
Right, that is what @wfurt mentioned offline.

Reply:
there's a FreeBSD image on the Azure Image Gallery so it should be pretty easy to create a Helix queue off of that.

Reply:
the issues we run into in the past are mostly not technical @akoeplinger.